apps/teams-test-app/src/components/privateApis/CopilotAPIs.tsx

```diff
@@ -23,11 +23,40 @@ const CopilotAPIs = (): ReactElement => {
       },
     });
 
+  const SendCustomTelemetryData = (): ReactElement =>
+    ApiWithTextInput<{
+      stageNameIdentifier: UUID;
+      timestamp: number;
+    }>({
+      name: 'sendCustomTelemetryData',
+      title: 'sendCustomTelemetryData',
+      onClick: {
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        validateInput: (_input) => {},
+        submit: async (input) => {
+          const result = await copilot.customTelemetry.sendCustomTelemetryData(
+            input.stageNameIdentifier,
+            input.timestamp,
+          );
+          return JSON.stringify(result);
+        },
+      },
+      defaultInput: JSON.stringify({
+        stageNameIdentifier: new UUID('805a4340-d5e0-4587-8f04-0ae88219699f'),
```

Comment:
Since the `UUID` constructor takes in a string input and this object is internal to our uses, I am of the opinion that we should try to keep the functions in the test app as agnostic towards how the internals of the SDK function. 
This way if changes happen to the internal UUID object, we don't break testing and if we do we will know it's probably for a good reason.

I am not committed to this and am open to hear what you and others think.

Reply:
The UUID object is now public, and I have exported it as part of this PR, so I think this should be okay now. We should not make changes to the UUID object that would cause anything to break. :)